packages/pancake-uikit/src/components/ButtonMenu/ButtonMenuItem.tsx

```diff
@@ -6,12 +6,12 @@ import { ButtonMenuItemProps } from "./types";
 
 interface InactiveButtonProps extends BaseButtonProps {
   forwardedAs: BaseButtonProps["as"];
-  colorKey: "primary" | "textSubtle";
+  colorkey: "primary" | "textSubtle";
 }
 
 const InactiveButton: PolymorphicComponent<InactiveButtonProps, "button"> = styled(Button)<InactiveButtonProps>`
   background-color: transparent;
-  color: ${({ theme, colorKey }) => theme.colors[colorKey]};
```

Comment:
The initial solution here is wrong imo. If you want to clean this, a better solution is to remove the `colorkey` attribute and compute the color based on the variant attribute

Reply:
@memoyil 

@RabbitDoge is right in it is not an ideal solution. Do you mind fixing it? No worries if you can't, I will get to it.

Reply:
@hachiojidev I will try to fix it :) if not I will ask your help

Reply:
@hachiojidev can you review it again?